README.md

```diff
@@ -1,277 +1,468 @@
+
+```ts
+import { BridgeFromEVMInput, EVMAddress, SDKNumber, KnownChainId, KnownTokenId } from "@brotocol-xyz/bro-sdk";
+// Choose your preferred web3 lib here
 import { ethers } from "ethers";
-  
-// Retrieve bridge information
-const bridgeInfo = await broSdk.bridgeInfoFromEVM({
-  fromChain: KnownChainId.EVM.Ethereum,
-  toChain: KnownChainId.Stacks.Mainnet,
-  fromToken: KnownTokenId.EVM.USDT,
-  toToken: KnownTokenId.Stacks.sUSDT,
-  amount: toSDKNumberOrUndefined(100),
-});
 
-console.log("Bridge Info:", bridgeInfo);
-  
 // Example signer setup using ethers.js
-const provider = new ethers.providers.JsonRpcProvider("https://mainnet.someprovider.io/YOUR_PROJECT_ID");
-const signer = new ethers.Wallet("SENDER_PRIVATE_KEY", provider);
+const provider = new ethers.JsonRpcProvider("https://mainnet.someprovider.io/YOUR_PROJECT_ID");
+const signer = new ethers.Wallet("000000000000000000000000000000000000000000000000000000000000002d", provider);
+const signerAddress = signer.address as `0x${string}`;
 
 const bridgeFromEVMInput: BridgeFromEVMInput = {
   fromChain: KnownChainId.EVM.Ethereum,
   toChain: KnownChainId.Stacks.Mainnet,
-  fromToken: KnownTokenId.EVM.USDT,
-  toToken: KnownTokenId.Stacks.sUSDT,
-  fromAddress: /* Sender EVM address */,
-  toAddress: /* Receiver Stacks principal */,
+  fromToken: evmToken as KnownTokenId.EVMToken,
+  toToken: stacksToken as KnownTokenId.StacksToken,
+  // Sender Ethereum address
+  fromAddress: signerAddress,
+  // Receiver Stacks principal
+  toAddress: "SP2ZD731ANQZT6J4K3F5N8A40ZXWXC1XFXHVVQFKE",
   amount: toSDKNumberOrUndefined(100),
-  sendTransaction: async (tx: 
-    {
-      from: EVMAddress /* Sender EVM address */
-      to: EVMAddress /* Bridge Endpoint address */
-      data: Uint8Array /* Transaction data */
-      recommendedGasLimit: SDKNumber /* Recommended gas limit */
-      value?: SDKNumber /* Transaction value */
-    }
-  ): Promise<{ txHash: string }> => {
+  sendTransaction: async (tx: {
+    from: EVMAddress; // Sender Ethereum address
+    to: EVMAddress; // Bridge Endpoint address
+    data: Uint8Array;
+    recommendedGasLimit: SDKNumber;
+    value?: SDKNumber;
+  }): Promise<{ txHash: string }> => {
     /**
      * Implementation for sending transaction on Ethereum mainnet
-     * See https://docs.ethers.org/v5/api/contract/contract/ for reference
+     * See https://docs.ethers.org/v6/ for reference
      */
     const txRequest = {
-    from: tx.from,
-    to: tx.to,
-    data: ethers.utils.hexlify(tx.data),
-    gasLimit: ethers.BigNumber.from(tx.recommendedGasLimit.split(" ")[0]), // Convert string to BigNumber
+      from: tx.from,
+      to: tx.to,
+      data: ethers.hexlify(tx.data),
+      // Convert SDKNumber into BigNumber
+      gasLimit: ethers.toBigInt(tx.recommendedGasLimit.split(" ")[0]), 
```

Comment:
I will add a new export `formatSDKNumber` to transform `toSDKNumberOrUndefined(123.33)` to `"123.33"`

This code is actually incorrect, the value of `tx.recommendedGasLimit` is actually `"123.33"` (although the type is `${string} (BroSDK number)`, I don't want users to mix SDKNumber with normal numeric string types, leading to incompatibilities in the future when our internal implementation changes)

Reply:
added https://github.com/Brotocol-xyz/bro-sdk/blob/39a0ecd2e45b19e5297d99b46a3c39194cfbb97c/src/sdkUtils/types.ts#L61-L63